packages/playground/src/weblets/micro_vm.vue

```diff
@@ -233,6 +233,11 @@ const images = [
     flist: "https://hub.grid.tf/tf-official-vms/nixos-micro-latest.flist",
     entryPoint: "/entrypoint.sh",
   },
+  {
+    name: "BTC Node Mycelium",
```

Comment:
do we have to have mycelium in the name?

Reply:
took it from flist name, but will confirm it